examples/sample-foundry-project/cannonfile.consumer.toml

```diff
@@ -38,19 +38,19 @@ target = ["greeters.greeter"]
 func = "changeCloneGreeting"
 args = ["<%= contracts.cloned.address %>", "<%= formatBytes32String('New Greeting!') %>"]
 
-extra.NewCloneGreeting.event = "NewGreetingAdded"
-extra.NewCloneGreeting.arg = 0
-extra.NewCloneGreeting.artifact = "Greeter"
+var.NewCloneGreeting.event = "NewGreetingAdded"
+var.NewCloneGreeting.arg = 0
+var.NewCloneGreeting.artifact = "Greeter"
 
-extra.OldCloneGreeting.event = "OldGreetingRemoved"
-extra.OldCloneGreeting.arg = 0
-extra.OldCloneGreeting.artifact = "Greeter"
+var.OldCloneGreeting.event = "OldGreetingRemoved"
+var.OldCloneGreeting.arg = 0
+var.OldCloneGreeting.artifact = "Greeter"
 
 # test to parse through previous emitted event values
 [invoke.set_new_greeting_for_next_clones]
 target = ["greeters.greeter"]
 func = "setGreeting"
-args = ["<%= extras.NewCloneGreeting %>"]
+args = ["<%= settings.NewCloneGreeting %>"]
```

Comment:
```suggestion
args = ["<%= var.NewCloneGreeting %>"]
```

Reply:
It is very important to match all instances of the full string because we are not dealing with programming language but with strings in toml and can only rely on full text search when performing refactoring.